internal/services/pubkey_service_test.go

```diff
@@ -51,3 +52,30 @@ func TestListPubkeysHandler(t *testing.T) {
 
 	assert.Equal(t, 2, len(pubkeys), "expected two pubkeys in response json")
 }
+
+func TestCreatePubkeyHandler(t *testing.T) {
+	ctx := context.Background()
+
+	store := []*models.Pubkey{}
+	ctx = dao_stubs.WithPubkeyDao(ctx, store)
+
+	values := map[string]interface{}{
+		"account_id": 1,
+		"name":       "very cool key",
+		"body":       "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEhnn80ZywmjeBFFOGm+cm+5HUwm62qTVnjKlOdYFLHN lzap",
+	}
```

Comment:
Very cool indeed, my 2021 key is on a roll! :-)

Reply:
You've asked for ir :P